src/JSONSchema/__tests__/formPathHandler.test.ts

```diff
@@ -0,0 +1,50 @@
+import { getObjectFromForm } from '../internal-path-handler'
+
+test('should return an object that matches the schema', () => {
+  const mockJSONSchema = {
+    type: 'object',
+    properties: {
+      firstName: {
+        type: 'string',
+      },
+      lastName: {
+        type: 'string',
+      },
+      address: {
+        type: 'object',
+        properties: {
+          city: {
+            type: 'string',
+          },
+          street: {
+            type: 'string',
+          },
+          streetNumber: {
+            type: 'integer',
+          },
+        },
+      },
+    },
+  }
+  const mockData = {
+    '#/firstName': 'Helena',
+    '#/lastName': 'Steck',
+    '#/address/city': 'RJ',
+    '#/address/street': 'Uma rua ae',
```

Comment:
English, please! And use a better example!

Reply:
Done